pagopa/pn-frontend-e2e
# E2E-api-call-implementation

## Short description

Added a new call to create a custom notification

## List of changes proposed in this pull request

- Created a new file to build the notification with the incoming specifications
- Added new client to perform the document preload
- Added new client to perform the document upload
- Created 2 new steps to create notifications for PF and PG

## How to test

Use the new steps to create a notification and verify that it is created correctly
